third_party/libusb/naclport/src/chrome_usb/types.cc

```diff
@@ -233,6 +298,20 @@ void EndpointDescriptorConverter::VisitFields(const EndpointDescriptor& value,
   callback(&value.extra_data, "extra_data");
 }
 
+template <>
+StructValueDescriptor<InterfaceDescriptor>::Description
+StructValueDescriptor<InterfaceDescriptor>::GetDescription() {
+  return Describe("chrome_usb::InterfaceDescriptor")
+      .WithField(&InterfaceDescriptor::interface_number, "interfaceNumber")
+      .WithField(&InterfaceDescriptor::alternate_setting, "alternateSetting")
+      .WithField(&InterfaceDescriptor::interface_class, "interfaceClass")
+      .WithField(&InterfaceDescriptor::interface_subclass, "interfaceSubclass")
+      .WithField(&InterfaceDescriptor::interface_protocol, "interfaceProtocol")
+      .WithField(&InterfaceDescriptor::description, "description")
+      .WithField(&InterfaceDescriptor::endpoints, "endpoints")
+      .WithField(&InterfaceDescriptor::extra_data, "extra_data");
```

Comment:
ditto

Reply:
Same as above - it's dictated by the naming in Chrome Extensions API.